p751toolbox/field_amd64.s

```diff
@@ -30,6 +30,15 @@
 #define P751X2_10  $0x1C25213F2F75B8CD
 #define P751X2_11  $0x0000DFCBAA83EE38
 
+DATA	P751P1_NZ<>+0x00(SB)/8,	$0xEEB0000000000000
```

Comment:
This is copy paste of values defined above

Reply:
I know. My guess was that it's generally faster to use the immediates, but that requires an additional auxiliary register and that may not be available in the reduction. I'll check tomorrow.